### ISSUE TITLE
Next/Previous buttons not disabled on initialization when there are not enough slides

### ISSUE THREAD

**Issue by nhardy** [benjamincharity/angular-flickity#80]
It looks like the `bc-flickity-previous` and `bc-flickity-next` directives do not initialize as disabled, even if there are not enough slides to scroll. It looks like the functions defined here: https://github.com/benjamincharity/angular-flickity/blob/master/src/previous/previous.directive.js#L71 and here: https://github.com/benjamincharity/angular-flickity/blob/master/src/next/next.directive.js#L73 need to be called when Flickity is initialized.

**Comment by benjamincharity** [comment]
Good catch @nhardy. I don't currently have much bandwidth to work on this, but will try to get to it soon(ish). As always, PRs are welcome!
